pkg/api/deprovision.go

```diff
@@ -77,6 +77,7 @@ func (s *server) deprovision(w http.ResponseWriter, r *http.Request) {
 		return
 	case service.InstanceStateProvisioned:
 	case service.InstanceStateProvisioningFailed:
+	case service.InstanceStateUpdatingFailed:
```

Comment:
i rebased off my other open PR and this came with it, so review this in the other one.